packages/primitives/heading/test/index.ts

```diff
@@ -0,0 +1,50 @@
+import test from 'tape'
+import { getAppPage } from 'foreal'
+
+const PACKAGE = '@primitives/heading'
+type TFixtures = 'h1'
+const getFixture = (filename: TFixtures) => `./fixtures/${filename}.tsx`
+
+test(`${PACKAGE} outputs children properly`, async (t) => {
+  const entryPointPath = require.resolve(getFixture('h1'))
+
+  const page = await getAppPage({
+    entryPointPath,
+  })
+
+  const h1Content = await page.$eval('#foo', (el: any) => el.textContent)
```

Comment:
Since we're only rendering one component, would it be possible to omit the id all-together and just target the first and only component that is rendered? 

Having an id that theoretically will be the same in every test feels very reduntant